build.gradle

```diff
@@ -1328,6 +1328,43 @@ project(':core') {
     duplicatesStrategy 'exclude'
   }
 
+  task libLicenseCheck(dependsOn: 'releaseTarGz', type: Copy) {
```

Comment:
I think it would be better to do this as part of releaseTarGz so we can't mistakenly produce an artifact that is missing licenses. I glanced through build.gradle and I don't see where we're gathering the libraries into the LICENSE file. Do you know where that happens?

Reply:
hi David, do you mean where we replace the content of LICENSE with the content of LICENSE-binary ? 
If that's the case, I think it should be below:
`from "$rootDir/NOTICE-binary" rename {String filename -> filename.replace("-binary", "")}`  in the releaseTarGz task